packages/ajv-validator/package.json

```diff
@@ -9,10 +9,10 @@
     "dist"
   ],
   "scripts": {
-    "test": "vitest --run",
-    "build": "tsup ./src/index.ts --format esm,cjs --dts",
-    "publint": "publint",
-    "release": "yarn build && yarn test && yarn publint && yarn publish"
```

Comment:
- [`vitest --run`](https://vitest.dev/guide/cli.html#run) disables watch mode, which is disabled by default in CI. The option is redundant, so I've removed it
- `tsup --format esm,cjs --dts` are both now referenced in the shared `tsup.config.ts` file, along with a few other options
- [`release`] - it doesn't look like this script is being used anywhere, Changesets doesn't look like it's configured to use it, so I thought is was best to remove it.

It looks like https://github.com/honojs/middleware/blob/main/.github/workflows/release.yml just runs `yarn build` and Changesets does the release. Using `prepack` should remove the need to build all workspaces before doing a publish, so only the packages being published are built